composer.json

```diff
@@ -12,7 +12,7 @@
 	"require": {
 		"php": "~7.1",
 		"phpstan/phpstan": "^0.10",
-		"dibi/dibi": "~3.0"
+		"nikic/php-parser": "^4.0"
```

Comment:
This extension isn't actually dependent on any symbol from PHP-Parser, but I'll allow it for the sake of consistency.